reactioncommerce/catalyst
# fix: Various component fixes

Resolves #147
Resolves #139
Impact: **minor**
Type: **bugfix**

## Changes
- Fix prop warnings for TextField component
- Add additional padding to TextField
- Fix padding on TableCell

## Testing
1. See no warnings in text field docs and the additional field padding
1. Check table components still look good. (the change was so minor it probably affects nothing)
